src/test/session-details-provider/session-details-app.js

```diff
@@ -38,3 +38,10 @@ process.on('SIGTERM', () => {
     process.exit(0)
   })
 })
+
+process.on('SIGINT', () => {
```

Comment:
I think this should be in a separate PR. It's quite a complicated PR anyway and this adds extra mental overhead.

Reply:
Removed, will update in it's own PR.